index.html

```diff
@@ -6,7 +6,170 @@
   </head>
   <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
+    <header class="page-header">
+      <nav class="navigation">
+
+        <a class="navigation-logo">
+          <img class="page-header-logo" alt="Логотип cайта туристического городка в штате Аризона." src="">
+        </a>
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <a class="navigation-link navigation-link-current">
+              Главная
+            </a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">
+              О Седоне
+            </a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">
+              Гостиницы
+            </a>
+          </li>
+        </ul>
+
+        <ul class="navigation-list navigation-user">
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">
+              <span class="visually-hidden">Поиск</span>
+            </a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">
+              <span class="visually-hidden">Избранное</span>
+              <span class="navigation-item-number-favorites">12</span>
+            </a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">Хочу сюда!</a>
+          </li>
+        </ul>
+
+      </nav>
+    </header>
+
+    <main class="main-index">
+      <h1 class="visually-hidden">Туристический городок Седона</h1>
+
+      <div class="hero">
+          <img class="hero-welcome" src="" alt="добро пожаловать">
+          <img class="hero-gorgeous" src="" alt="в великолепную">
+          <img class="hero-sedona" src="" alt="Судону. Потому что Гранд-Каньон отстой!">
+          <img class="hero-devider" src="">
+      </div>
```

Comment:
Не исправил опечатку.
И еще отступ тут 4 пробела, а у тебя должно быть 2 во всей разметке. Это регулируется в editorconfig, при нажатии кнопки tab - у тебя автоматически должен прыгнуть на 2 пробела
```suggestion
      <div class="hero">
        <img class="hero-welcome" src="" alt="добро пожаловать">
        <img class="hero-gorgeous" src="" alt="в великолепную">
        <img class="hero-sedona" src="" alt="Седону. Потому что Гранд-Каньон отстой!">
        <img class="hero-devider" src="">
      </div>
```

Reply:
исправил